ruby/wipe-appwrite-bucket/README.md

```diff
@@ -0,0 +1,50 @@
+# 🗑 Wipe Appwrite Bucket
+
+A Ruby cloud function to wipe a complete bucket in Appwrite by passing the bucket id as a payload.
+
+_Example input:_
+
+```json
+{
+  "bucketId":"profilePictures"
+}
+```
+
+_Example output:_
+
+```json
+{
+  "success":true
+}
+```
+
+_Example error output:_
+
+```json
+{
+  "success":false,
+  "message":"bucketId was not provided."
+}
+```
+
+## 📝 Environment Variables
+
+List of environment variables used by this cloud function:
```

Comment:
Would you please remove references to "environment". Appwrite variables used to be environment variables, but they aren't anymore.

Reply:
Updated.